version.props

```diff
@@ -1,11 +1,15 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>2.1.0</VersionPrefix>
-    <ExperimentalProjectVersionPrefix>0.4.0</ExperimentalProjectVersionPrefix>
     <VersionSuffix>preview1</VersionSuffix>
+    <ExperimentalVersionPrefix>0.4.0</ExperimentalVersionPrefix>
```

Comment:
Can you leave as ExperimentalProjectVersionPrefix? This is what we are using in other repos, and it matches what our update scripts will look for.

Reply:
example: https://github.com/aspnet/Universe/blob/release/2.0/scripts/PatchVersionPrefix.ps1#L45

Reply:
I went off [logging](https://github.com/aspnet/Logging/blob/dev/version.props#L5) like you said. Do we need to update logging? What's another example?

Reply:
Ok I'll change it, but we should circle back and fix logging and anywhere else with the wrong name.

Reply:
Oh, I didn't realize logging was wrong too. Yeah, let's make sure we're consistent. Now I wonder what is in release/2.0. I thought we had used ExperimentalProjectVersionPrefix everywhere